rust/Cargo.lock

```diff
@@ -0,0 +1,21 @@
+[root]
+name = "suricata"
+version = "4.0.0-dev"
```

Comment:
Doing it as a .in file works, the main problem being that our current version, "4.0dev" will cause Cargo to error out:
```
Caused by:
  cannot parse '4.0' as a semver for key `package.version`
```
The closest thing to our version that will parse is 4.0.0-dev.